substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -257,6 +257,7 @@ private static <T> String oR(OptionKey<T> option) {
     static final String oHNumberOfThreads = oH(NativeImageOptions.NumberOfThreads);
 
     final Map<String, String> imageBuilderEnvironment = new HashMap<>();
+    final ArrayList<String> bundleLauncherArgs = new ArrayList<>();
```

Comment:
Please move this new field into `BundleSupport` and change the above code in `APIOptionHandler.java` to use
```java
if (nativeImage.useBundle() && option.launcherOption) {
...
   nativeImage.bundleSupport.bundleLauncherArgs.add(...);
...
}
```

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/939e77cd7c6373a307fb5f85a89ac3a5d9af9ee9